values.yaml

```diff
@@ -0,0 +1,45 @@
+clusterDomain: "se6099.opencraft.hosting"
+
+ingress-nginx:
+  enabled: true
+  controller:
+    config:
+      proxy-body-size: 100m
+
+cert-manager:
+  enabled: false
+  installCRDs: true
+  email: "gabor@opencraft.com"
+
+elasticsearch:
+  enabled: false
+
+metricsserver:
+  enabled: false
+
+vpa:
+  enabled: false
+
+opensearch:
+  enabled: false
+
+prometheusstack:
+  enabled: false
+
+k8sdashboard:
+  enabled: true
+
+openfaas:
+  enabled: true
+  ingress:
+    enabled: true
+    hosts:
+      - host: openfaas.se6099.opencraft.hosting  # Replace with gateway.example.com if public-facing
+        serviceName: gateway
+        servicePort: 8080
+        path: /
+    ingressClassName: nginx
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
[question] Is it possible to use OpenFaas without cert manager or if not should we state that dependency somewhere, possibly in https://github.com/openedx/openedx-k8s-harmony?tab=readme-ov-file#technology-stack-and-architecture?

Reply:
It could be used without cert-manager as cert-manager is optional. It was for my testing. The whole file should not have been committed.